getDeployConfig.js

```diff
@@ -7,7 +7,7 @@ const fs = require('fs')
 const os = require('os')
 const projectRoot = require('app-root-path')
 const appPackage = require(projectRoot.path + '/package.json')
-const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployFile: ['df'] } })
+const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployConfigFile: [ 'df'] } })
```

Comment:
```suggestion
const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployConfigFile: ['df'] } })
```

Don't think we need this extra space. Commit it if you agree

Reply:
This commit suggestion thing is awesome